.github/workflows/deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Deployment
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
can you not remove this?

Reply:
it is used to call the workflow manually